src/main/java/com/epam/api/services/CommonService.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.api.services;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+import java.util.Map;
+
+import static com.epam.api.services.PropertyService.getValue;
+import static io.restassured.RestAssured.given;
+
+public class CommonService {
+    private RequestSpecification REQUEST_SPECIFICATION;
+
+    public CommonService() {
+        REQUEST_SPECIFICATION = new RequestSpecBuilder()
+                .addQueryParam("key", getValue("API-Key"))
```

Comment:
create methods in property resder class getKey() and getToken()

Reply:
+